### PR TITLE
Added very basic content to Events page

### PR DESCRIPTION
**Pull request by tbg510**
## Issue Number
Works on Issue #149 
## Description
Added very basic static content to the Events page. Currently it just has two separate categories for Today's Events and Upcoming Events each with a list of events. Events have a date, title, and description. In the future, I plan on creating components to make adding and listing events easier. I just wanted to get some basic content onto the page for now.
## Testing
I edited the test for the Events page so that it will pass. Right now it just checks if "Today's Events" is present. When I finish the page completely the test will have to be changed to cover everything.
## Possible Errors
No errors since it's static content. I'm open to suggestions for event content or overall formatting, however.
## Images
image
Current content for Events page.

### CODE REVIEW

**Review by junxjr — APPROVED** [review]
The event's page looks very simple and clear. I see that all the data are in the index.js, I would suggest if there's time, maybe we can move it to the data folder so it keeps the same format as the other pages.

**Comment by tbg510** [comment]
> The event's page looks very simple and clear. I see that all the data are in the index.js, I would suggest if there's time, maybe we can move it to the data folder so it keeps the same format as the other pages.

Yea I agree, I was planning on working on separating the data and components into their own files next.

File: small-talk/pages/events/index.js [hunk 1]
```diff
@@ -14,7 +14,55 @@ export const Events = ({data}) =>{
     return(
         <div className='bg-slate-800 text-white'>
             <Layout width={windowWidth} renderHeader={true} renderFooter={true}>
-                <h1>Events Page</h1>
+                <h1 className = 'font-bold'>Today&apos;s Events</h1>
+                <ul className = 'divide-y divide-solid'>
+                    <li className = 'list-disc'>
+                        <ul>
+                            <li>2024-5-2</li>
+                            <li>Toy Drive</li>
```
Comment (aiharc1) [inline small-talk/pages/events/index.js:22]: Only criticism is there is no use of templates for the events. The events are "hard coded" and have the possibility of being of non-uniform structure. 

Maybe if there's time we can implement a list of uniform templates with unique event data down the line

File: small-talk/pages/events/index.js [hunk 2]
```diff
@@ -14,7 +14,55 @@ export const Events = ({data}) =>{
...
+                        <ul>
+                            <li>2024-5-2</li>
+                            <li>Meetup with Timmy</li>
+                            <li>Timmy and I will hangout together and play Roblox!!</li>
+                        </ul>
+                    </li>
+                </ul>
+
+                <h1 className = 'font-bold pt-10'>Upcoming Events</h1>
+                <ul className = 'divide-y divide-solid'>
```
Comment (aiharc1) [inline small-talk/pages/events/index.js:36]: The lines of words look a little bunched up. Maybe divide by a line + an extra line if possible?

**Review by aiharc1 — APPROVED** [review]
Looks good for the first itteration of the events page. Just a few criticisms about the lack of template use for the events and adding a space between each event would help with readability. It would also be cool if we could throw in some details to make it more appealing to the eye. Nice work!

File: small-talk/pages/events/index.js [hunk 3]
```diff
@@ -14,7 +14,55 @@ export const Events = ({data}) =>{
...
+                    <li className = 'list-disc'>
```
Comment (39otsu) [inline small-talk/pages/events/index.js:37]: This looks like a good opportunity to improve upon this to make it into a component, and feed data into it from a database

**Review by 39otsu — APPROVED** [review]
Looks clean! Great job with uniformity

**Pull request merged**